src/js/view/controls/components/volumetooltip.js

```diff
@@ -23,7 +23,6 @@ export default class VolumeTooltip extends Tooltip {
 
         this.volumeSlider.on('update', function (evt) {
             this.trigger('update', evt);
-            this.el.focus();
```

Comment:
Why can this be safely removed?

Reply:
Before, when the slider was interacted with, it would push focus back onto the volume icon. Before the changes for 8.7, the slider would still be visible. Now, it's not so we don't want to push focus back on to the volume icon - we want to keep it on the slider.